From bdd555b0bca45515b2a930017ef6df7616ee4f58 Mon Sep 17 00:00:00 2001
From: Larry Finger <Larry.Finger@lwfinger.net>
Date: Sat, 15 Mar 2014 12:06:42 -0500
Subject: [PATCH 262/390] Remove
 CONFIG_UPDATE_INDICATE_SEQ_WHILE_PROCESS_ADDBA_REQ

This variable is not needed any longer.

Signed-off-by: Larry Finger <Larry.Finger@lwfinger.net>
---
 core/rtw_wlan_util.c | 21 ++-------------------
 1 file changed, 2 insertions(+), 19 deletions(-)

--- a/drivers/staging/rtl8192du/core/rtw_wlan_util.c
+++ b/drivers/staging/rtl8192du/core/rtw_wlan_util.c
@@ -1485,38 +1485,21 @@ int update_sta_support_rate(struct rtw_a
 void process_addba_req(struct rtw_adapter *padapter, u8 *paddba_req, u8 *addr)
 {
 	struct sta_info *psta;
-	u16 tid;
-#ifdef CONFIG_UPDATE_INDICATE_SEQ_WHILE_PROCESS_ADDBA_REQ
-	u16 start_seq;
-#endif
-	u16 param;
 	struct recv_reorder_ctrl *preorder_ctrl;
 	struct sta_priv *pstapriv = &padapter->stapriv;
 	struct ADDBA_request	*preq = (struct ADDBA_request *)paddba_req;
 	struct mlme_ext_priv	*pmlmeext = &padapter->mlmeextpriv;
 	struct mlme_ext_info	*pmlmeinfo = &(pmlmeext->mlmext_info);
+	u16 tid;
+	u16 param;
 
 	psta = rtw_get_stainfo(pstapriv, addr);
 
 	if (psta) {
-		#ifdef CONFIG_UPDATE_INDICATE_SEQ_WHILE_PROCESS_ADDBA_REQ
-		start_seq = le16_to_cpu(preq->BA_starting_seqctrl) >> 4;
-		#endif
 		param = le16_to_cpu(preq->BA_para_set);
 		tid = (param>>2)&0x0f;
 
 		preorder_ctrl = &psta->recvreorder_ctrl[tid];
-
-		#ifdef CONFIG_UPDATE_INDICATE_SEQ_WHILE_PROCESS_ADDBA_REQ
-		preorder_ctrl->indicate_seq = start_seq;
-		#ifdef DBG_RX_SEQ
-		DBG_8192D("DBG_RX_SEQ %s:%d IndicateSeq: %d, start_seq: %d\n", __func__, __LINE__,
-			  preorder_ctrl->indicate_seq, start_seq);
-		#endif
-		#else
-		preorder_ctrl->indicate_seq = 0xffff;
-		#endif
-
 		preorder_ctrl->enable = (pmlmeinfo->bAcceptAddbaReq == true) ? true : false;
 	}
 }
